From b9a4991b4f8dd510bad447a1eb5227549f661af4 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Fri, 15 Apr 2022 10:18:45 -0700
Subject: [PATCH] Add mising header found with musl

Fixes
../../sandbox/linux/services/credentials.h:41:25: error: unknown type name 'uid_t'; did you mean 'pid_t'?
  static bool GetRESIds(uid_t* resuid, gid_t* resgid);
                        ^~~~~
                        pid_t

Upstream-Status: Pending
Signed-off-by: Khem Raj <raj.khem@gmail.com>

---
 sandbox/linux/services/credentials.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/sandbox/linux/services/credentials.h b/sandbox/linux/services/credentials.h
index e59ebef4a2..658a60115c 100644
--- a/sandbox/linux/services/credentials.h
+++ b/sandbox/linux/services/credentials.h
@@ -16,6 +16,8 @@
 #include <string>
 #include <vector>
 
+#include <sys/types.h>
+
 #include "sandbox/linux/system_headers/capability.h"
 #include "sandbox/sandbox_export.h"
 
